.env

```diff
@@ -2,7 +2,7 @@
 
 if [ -f "requirements.txt" ]; then
     if [ ! -d "venv" ]; then
-        virtualenv venv
+        virtualenv -p /usr/bin/python2.7 venv
```

Comment:
What about ``virtualenv -p `which python` venv`` ?
I would not assume the python being used to be on the standard place.